src/range.rs

```diff
@@ -188,6 +188,9 @@ indexed_range_impl!{isize}
 unindexed_range_impl!{u64}
 unindexed_range_impl!{i64}
 
+// 128-bit support uses a feature to permit rustc < 1.26
+#[cfg(feature = "range128")] unindexed_range_impl !{u128}
+#[cfg(feature = "range128")] unindexed_range_impl !{i128}
```

Comment:
Notice that this macro defines a `fn len(&self) -> u64`, which may overflow for 128-bit ranges.  I think we can just add a second macro parameter for the type of unsigned length, using `u64` for the 64-bit ranges and `u128` for the 128-bit ranges.  A unit test for these lengths would be good too.